_pages/index.md

```diff
@@ -7,6 +7,7 @@ layout: homepage
 
 # Hero
 hero_image: 'img/fluids_blend.png'
+attribution: '<strong>Image Credit:</strong> Matthew Larsen, LLNL. This image is of an idealized Inertial Confinement Fusion (ICF) simulation of a Rayleigh-Taylor instability with two fluids mixing in a spherical geometry.'
```

Comment:
Should this be named something like `hero_attribution` to make it clear what is being attributed?